nixos/modules/security/duosec.nix

```diff
@@ -24,22 +24,6 @@ let
     motd=${boolToStr cfg.motd}
     accept_env_factor=${boolToStr cfg.acceptEnvFactor}
   '';
-
-  loginCfgFile = optionalAttrs cfg.ssh.enable {
-    "duo/login_duo.conf" =
-      { source = pkgs.writeText "login_duo.conf" configFileLogin;
-        mode   = "0600";
-        user   = "sshd";
-      };
-  };
-
-  pamCfgFile = optional cfg.pam.enable {
```

Comment:
Uhm, `loginCfgFile` is an attrset but the `optional` here is producing a list. It shouldn't be an error yet: `loaOf` is deprecated but it should just raise a warning, not sure what's going on here.
In any case this is the proper fix for the future removal of `loaOf`.

Reply:
Yeah... wild stab in the dark. Not worth effort as `master` is different anyways. Thanks for taking a look!